tests/sentry/issues/test_ingest.py

```diff
@@ -164,3 +178,35 @@ def test(self) -> None:
             "location": event.location,
             "last_received": event.datetime,
         }
+
+
+@region_silo_test
+class SaveIssueOccurrenceToEventstreamTest(OccurrenceTestMixin, TestCase):  # type: ignore
+    def test(self) -> None:
+        # TODO: We should make this a platform event once we have one
+        event = self.store_event(data={}, project_id=self.project.id)
```

Comment:
do you want me to add a helper method to grab a generic ingestion event?

Reply:
Yeah I think that'd be helpful. We can just add a fixture [here](https://github.com/getsentry/sentry/blob/6c19ea2ebec191f175456bfdeafd8c427883ad14/src/sentry/data/samples/transaction-n-plus-one.json#L1) and load with `load_data`

Reply:
will add this in a separate PR